pythoncode/antDongle.py

```diff
@@ -1569,15 +1572,62 @@ def msgUnpage49_TargetPower(info):
     format = sc.no_alignment + fChannel + fDataPageNumber + fReserved + fTargetPower
     tuple  = struct.unpack (format, info)
 
-    rtn = tuple[nTargetPower] / 4      # returns units of 1Watt
+    TargetPower = tuple[nTargetPower] / 4   # returns units of 1Watt
 
-    return rtn
+    return TargetPower
+
+# ------------------------------------------------------------------------------
+# P a g e 5 0   W i n d R e s i s t a n c e

```

Comment:
I will try to do some SimulANT testing you later on, if you haven't already.

Reply:
I see you refer to some code-lines, but do not understand why

Reply:
The addition of ANT+ page 50 and 51 parameters into the Grade2Power() gribble calcs... as you copy-pasted, I will use SimulANT to send specifically crafted pages to FortiusANT in order to test the page parsing and the wiring through to the calculation. I put this comment against the header line that introduces the new ANT+ page to indicate it applies to the addition as a whole.